src/sql/base/browser/ui/editableDropdown/dropdown.ts

```diff
@@ -70,7 +70,7 @@ const defaults: IDropdownOptions = {
 	maxHeight: 300,
 	errorMessage: errorMessage,
 	contextBorder: Color.fromHex('#696969'),
-	actionLabel: nls.localize('dropdownAction.toggle', "Toggle dropdown")
```

Comment:
What are these `actionLabel`s for? Were they added for accessibility reasons?

Reply:
Looking at the git blame for these they were definitely added in an accessibility commit by @anthonydresser. Anthony do you remember if we need these labels for accessibility? I'd recommend keeping them so we don't accidentally reintroduce accessibility bugs.

Reply:
We should probably be using the aria-label for that instead then - the tooltip showing up is weird and not normal for dropdowns.

Reply:
Yeah you shouldn't remove this; this is an accessibility requirement for screen readers that the down arrow states its action.

Reply:
But there shouldn't be any issue with using aria-label right? That's what we use in most of the rest of the product for accessibility so using the tooltip here seems odd.

Reply:
Alright, so testing this out the ideal solution is to use aria-label. Unfortunately the actions are not hooked up in a way that we can directly add that attribute - it's a separate VS class that creates the actual elements and it doesn't support aria-label. I could do some DOM manipulation to set it but I'd avoid doing hacky stuff like that - especially since it works fine right now (it's just a little odd to have the tooltip there).